pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -55,6 +56,11 @@ buildPythonPackage rec {
 
   checkInputs = [ cacert ] ++ optional-dependencies.DNSSEC;
 
+  # don't run live tests
+  env = lib.optionalAttrs stdenv.hostPlatform.isDarwin {
+    NO_INTERNET = 1;
```

Comment:
Just do it on all of them?

Reply:
I thought triggering linux rebuilds would be bad at this point in `staging-next` and I think that on linux this function will return false anyway. but can make it unconditional if the rebuilds don't matter.

```python
def is_internet_reachable():
    """Check if the Internet is reachable.

    Setting the environment variable `NO_INTERNET` will let this
    function always return False. The result is cached.

    We check using the Google and Cloudflare public resolvers as they are highly
    available and have well-known stable addresses.
    """
    global _internet_reachable
    if _internet_reachable is None:
        if os.environ.get("NO_INTERNET"):
            _internet_reachable = False
        else:
            global _have_ipv4
            _have_ipv4 = check_networking(["8.8.8.8", "1.1.1.1"])
            global _have_ipv6
            _have_ipv6 = check_networking(
                ["2001:4860:4860::8888", "2606:4700:4700::1111"]
            )
            _internet_reachable = _have_ipv4 or _have_ipv6
    return _internet_reachab
```

Reply:
It is mostly about just doing the simple thing, but yeah, the rebuilds could be an issue. Make it conditional for now and remove the conditional in staging-not-next?

Reply:
ok. has condition again.